src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -99,6 +100,10 @@ public class KplMessageHandler extends AbstractAwsMessageHandler<Void> implement
 
 	private volatile ScheduledFuture<?> flushFuture;
 
+	private long maxRecordsInFlight = 0;
```

Comment:
Can we name this instead like `maxInFlightRecords`?

Reply:
renamed.